src/Resources/views/root.html.twig

```diff
@@ -0,0 +1,5 @@
+{#- @var \Becklyn\Menu\Item\MenuItem root -#}
```

Comment:
```suggestion
{#- @var root \Becklyn\Menu\Item\MenuItem -#}
```

Reply:
That is incorrect. The order is always 

1. `@var`
2. type
3. variable name

See the reference https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#824-var or basically any other phpdoc:

![2019-07-08 at 13 44](https://user-images.githubusercontent.com/1032411/60807655-8759a700-a186-11e9-8c2a-a0a48cd18cbd.png)

Reply:
Ah well, I've always used the version I've suggested. Since neither version is recommended (at least I couldn't find it anywhere), I'm fine with it :)

![image](https://user-images.githubusercontent.com/439899/60808252-1d420180-a188-11e9-9dee-0e0457df6c67.png) 
![image](https://user-images.githubusercontent.com/439899/60808258-1f0bc500-a188-11e9-8e7e-aa7ee5dedbb7.png)

Reply:
As this is basically PHPDoc we should stick to the PHPDoc ordering, which is as I described.

We should keep it consistent in our projects as well 👍